csharp/ql/src/experimental/Security Features/campaign/Solorigate-Readme.md

```diff
@@ -0,0 +1,121 @@
+# Working with Solorigate queries
```

Comment:
Could I suggest that you include some information about the original authors (and their affiliation) of these queries, and some external references? For example: https://aka.ms/solorigate, and as soon as you have a URL for your blog post that'd be great to include too.

Reply:
Does the intro cover what you were looking for?  BTW, the aka addresses don't yet point to final destinations - but I'll update them as soon as the blog is published.